### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jperata**
Relates to https://github.com/bespoken/bst/issues/505

### CODE REVIEW

**Comment by jperata** [comment]
@ecruzado at the moment this is for the include/exclude tags and the override options with an example.
Add a ticket for the other ones not documented and we will assign the time to update them.

File: end-to-end/guide.md [hunk 1]
```diff
@@ -109,6 +109,7 @@ Below the end-to-end testing configuration options and what they do are listed:
 * platform - The platform that is being tested - can be either `alexa` or `google` - defaults to `alexa`
 * type - The type of test being run - can be either `unit` or `e2e` - defaults to `unit`
 * [trace](#viewing-response-payloads) - Causes request and response JSON payloads from the skill to be printed to the console
+* [include and exclude](#including-or-excluding-tests-using-tags) - Runs or Skip the tests having the particular indicated tags
```
Comment (jkelvie) [inline end-to-end/guide.md]: Please put this in alphabetical order (as the other ones are)

File: end-to-end/guide.md [hunk 2]
```diff
@@ -446,6 +461,39 @@ module.exports = {
 
 The filter is a very useful catch-all for handling tricky test cases that are not supported by the YAML test syntax or if you want to fine tune some aspects of the tests.
 
+## Including or excluding tests using tags
+
+By specifying tags in particular tests you can then run only the tests you want. Let's say you have tests specific to the first time a user use your skill, we are going to apply the tag "FirstUse" to them:
```
Comment (jkelvie) [inline end-to-end/guide.md]: "first time a user **uses** your skill"

File: end-to-end/guide.md [hunk 3]
```diff
@@ -446,6 +461,39 @@ module.exports = {
...
+
+```
+---
+- test: open the skill
+- tags: FirstUse
```
Comment (jkelvie) [inline end-to-end/guide.md]: Can you show two tags just to clarify how they are specified? "FirstUse, Alexa"

File: end-to-end/guide.md [hunk 4]
```diff
@@ -446,6 +461,39 @@ module.exports = {
...
+- open my skill: hello
+```
+
+If you want to run all the tests that have that particular tag, you can edit testing.json to indicate that those are the ones to run by adding the "include" property:
```
Comment (jkelvie) [inline end-to-end/guide.md:477]: "Note that multiple tags can be applied to a test, as a comma-delimited list."

File: end-to-end/guide.md [hunk 5]
```diff
@@ -446,6 +461,39 @@ module.exports = {
...
+
+You can also use the exclude property to prevent some tests from being run, suppose we marked some broken tests with the tag broken, the following configuration will prevent those tests marked from being run:
+
+```
+{
+    "exclude": ["broken"],
+}
+```
+
+Remember you can also use the override properties when executing the bst test command and also combine include and exclude together. This command will run all the tests that have either FirstUse or ReturningUser tags but exclude the ones that are also marked as broken
```
Comment (jkelvie) [inline end-to-end/guide.md:493]: Awesome functionality (and good docs too) :-)

**Review by jkelvie — CHANGES_REQUESTED** [review]

File: end-to-end/guide.md (hunk 4 above)
Comment (jperata) [reply end-to-end/guide.md:477]: included after the example

File: end-to-end/guide.md [hunk 6]
```diff
@@ -105,6 +105,7 @@ Below the end-to-end testing configuration options and what they do are listed:
 * [filter](#filtering-during-test) - The (optional) path to a class that can be used to override value on the request and response
 * [findReplace](#findreplace) - Values that will be replaced in the scripts before execution
 * [homophones](#homophones) - Values that will be replaced in actual responses from the virtual device
+* [include and exclude](#including-or-excluding-tests-using-tags) - Runs or Skip the tests having the particular indicated tags
```
Comment (jkelvie) [inline end-to-end/guide.md]: "having the **specified** tags"

File: end-to-end/guide.md [hunk 7]
```diff
@@ -446,6 +461,41 @@ module.exports = {
...
 The filter is a very useful catch-all for handling tricky test cases that are not supported by the YAML test syntax or if you want to fine tune some aspects of the tests.
 
+## Including or excluding tests using tags
+
+By specifying tags in particular tests you can then run only the tests you want. Let's say you have tests specific to the first time a user uses your skill, we are going to apply the tag "FirstUse" to them:
+
+```
+---
+- test: open the skill
+- tags: FirstUse,Alexa
```
Comment (jkelvie) [inline end-to-end/guide.md]: Add a space after the comma

**Review by jkelvie — CHANGES_REQUESTED** [review]

**Review by jkelvie — APPROVED** [review]

**Pull request merged**